app/Services/Email.php

```diff
@@ -133,8 +134,6 @@ protected function parseLinks($string)
      */
     protected function setupEmail()
     {
-        $leaderboardVars = $this->getLeaderboardVars($this->users);
```

Comment:
oh ok, i just ended up adding a second if(is leaderboard email) but sounds like rebasing would just be cleaner